Assignments/Program.cs

```diff
@@ -1,9 +1,10 @@
 ﻿// ---------------------------------------------------------------------------------------
-// T13 - Strong Password
+// T15 - Isogram
 // ---------------------------------------------------------------------------------------
 using System;
+using System.Collections;
 using System.Collections.Generic;
-using System.Linq;
+using System.Globalization;

```

Comment:
![image](https://github.com/vikrampg1710-tmm/Training/assets/75829713/a2d63da0-a7d5-4f38-9cba-3f22a917c90d)
Unnecessary usings to be removed